e2e-tests/commands/shared-commands/assertions/assert-empty-contract-completion-date-field-values.js

```diff
@@ -12,9 +12,9 @@ const {
  * Assert all CONTRACT_COMPLETION_DATE field values are empty.
  */
 const assertEmptyContractCompletionDateFieldValues = () => {
-  cy.checkValue(field(CONTRACT_COMPLETION_DATE).dayInput(), '');
-  cy.checkValue(field(CONTRACT_COMPLETION_DATE).monthInput(), '');
-  cy.checkValue(field(CONTRACT_COMPLETION_DATE).yearInput(), '');
+  field(CONTRACT_COMPLETION_DATE).dayInput().should('have.value', '');
```

Comment:
the checkValue command?

Reply:
It does not work for date fields. It relies on an `.input` method, which is irrelevant for date fields (`dayInput` etc)

Need to create a new command for this.

I am not sure how this change got merged with failing tests, seems GHA did not correctly re-run some tests or something in [this PR](https://github.com/UK-Export-Finance/exip/pull/3141)

Reply:
Have created EMS-3898 to address this.